slrealizer/make_lens_source_object.py

```diff
@@ -0,0 +1,34 @@
+import os
```

Comment:
Can you add a suitable header to this script, please? It should include:  a one-line description of its Purpose, any Comments you want to make, what its Inputs and Outputs are, and who its Author is (and how to contact them - ie via an issue on the SLRealizer repo). In the comments you shoudl explain the hard-coded choices, in terms of the sample you want to make and why you want to make it.

Reply:
Done in `demo/Example+SLRealizer+Usage.ipynb`